CONTRIBUTING.rst

```diff
@@ -48,6 +48,10 @@ Then you can run the testsuite with::
 
     py.test
 
+- Note: Werkzeug must also be installed for all tests to pass::
```

Comment:
Sorry for doing this review chunked.

I think this note should be after the first paragraph where virtualenv is linked, so the steps the dev has to perform are in order.